### PR TITLE
add: `tdrop-git`

### PR DESCRIPTION
**Pull request by thomascrha**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [x] Contact devs
- [x] Add maintainer to pacscript

### CODE REVIEW

**Comment by thomascrha** [comment]
Sorry about that - I think this is now all ready to go - I have no idea how that `-ng` got there but it did. Also thanks for making this - its pretty sweetones.

**Comment by Elsie19** [comment]
sweetones?

File: packages/tdrop-git/tdrop-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,18 @@
+maintainer="Thomas Crha <thomas@9bitbyte.com>"
+
```
Comment (Elsie19) [inline packages/tdrop-git/tdrop-git.pacscript:2]: Put this near the end of the variables, it's rarely at the beginning.

File: packages/tdrop-git/tdrop-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+name="tdrop-git"
+gives="tdrop"
+url="https://github.com/noctuid/tdrop.git"
+pkgver="0.5.0"
+homepage='https://github.com/noctuid/tdrop'
+pkgdesc="A Glorified WM-Independent Dropdown Creator"
+makedepends=("cmake")
+depends=("xdotool" "bash" "gawk" "procps")
```
Comment (Elsie19) [inline packages/tdrop-git/tdrop-git.pacscript:10]:
```suggestion
depends=("xdotool" "gawk" "procps")
```
Already installed on debian and ubuntu by default.

**Review by Elsie19 — APPROVED** [review]

**Comment by thomascrha** [comment]
sorry, sweetones is an Australian Colloquialism - replace it with awesome.

File: packages/tdrop-git/tdrop-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+pkgver="0.5.0"
+homepage='https://github.com/noctuid/tdrop'
+pkgdesc="A Glorified WM-Independent Dropdown Creator"
+maintainer="Thomas Crha <thomas@9bitbyte.com>"
+makedepends=("cmake")
+depends=("xdotool" "gawk" "procps")
+
+package() {
+  sudo install -Dm755 "tdrop" "/usr/local/bin/tdrop"
+  sudo install -Dm644 "tdrop.1" "/usr/local/share/man/man1/tdrop.1"
```
Comment (Elsie19) [inline packages/tdrop-git/tdrop-git.pacscript:13]: Can you compress this first before installing it?

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/tdrop-git/tdrop-git.pacscript (hunk 3 above)
Comment (thomascrha) [reply packages/tdrop-git/tdrop-git.pacscript:13]: What do you mean by compress? Tar/zip it ? Could you show me an example of another pacscript that does what you are describing

Comment (thomascrha) [reply packages/tdrop-git/tdrop-git.pacscript:13]: Or do you want to combine the command into one ?

Comment (Elsie19) [reply packages/tdrop-git/tdrop-git.pacscript:13]: Sure! Basically manpages are to be compressed in the gz format with max compression. So above line 13, you would put something like:
```bash
gzip -9 tdrop.1
```
Then on line 13, you can `sudo install -Dm644 "tdrop.1.gz" -t "/usr/local/share/man/man1/"`

Comment (thomascrha) [reply packages/tdrop-git/tdrop-git.pacscript:13]: I created a prepare step for the gzip compress - similar to the example.

File: packages/tdrop-git/tdrop-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,20 @@
+maintainer="Thomas Crha <thomas@9bitbyte.com>"
```
Comment (Elsie19) [inline packages/tdrop-git/tdrop-git.pacscript:1]: Put it where you had it before.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/tdrop-git/tdrop-git.pacscript (hunk 4 above)
Comment (thomascrha) [reply packages/tdrop-git/tdrop-git.pacscript:1]: sorry about that

File: packages/tdrop-git/tdrop-git.pacscript [hunk 5]
```diff
@@ -0,0 +1,20 @@
+name="tdrop-git"
+gives="tdrop"
+url="https://github.com/noctuid/tdrop.git"
+pkgver="1.5.0"
+homepage="https://github.com/noctuid/tdrop"
+pkgdesc="A Glorified WM-Independent Dropdown Creator"
+maintainer="Thomas Crha <thomas@9bitbyte.com>"
+
```
Comment (Elsie19) [inline packages/tdrop-git/tdrop-git.pacscript:8]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**